lib/util/assertString.js

```diff
@@ -3,12 +3,28 @@
 Object.defineProperty(exports, "__esModule", {
   value: true
 });
+
+var _typeof = typeof Symbol === "function" && typeof Symbol.iterator === "symbol" ? function (obj) { return typeof obj; } : function (obj) { return obj && typeof Symbol === "function" && obj.constructor === Symbol && obj !== Symbol.prototype ? "symbol" : typeof obj; };
```

Comment:
Could make the function a one-liner arrow function? We were progressively moving to ES6:

```javascript
... ? (obj) => typeof obj;
```

For better readability (line-length), I'd suggest this rewrite:

```javascript
const _typeof = typeof Symbol === "function" && typeof Symbol.iterator === "symbol" ? 
  obj => typeof obj : 
  obj => obj 
    && typeof Symbol === "function"
    && obj.constructor === Symbol
    && obj !== Symbol.prototype ?
      "symbol" : typeof obj;
```

Reply:
Sure, that file I believe was generated automatically when I ran tests but I can just change it to that manually.

Reply:
Oops, my bad. Don't change manually here, but it's original location is `src/`

Reply:
Here - https://github.com/chriso/validator.js/blob/master/src/lib/util/assertString.js @DCtheTall